scripts/ecs-anywhere-install.sh

```diff
@@ -27,6 +27,22 @@ if [ -f "/sys/fs/cgroup/cgroup.controllers" ]; then
     fail
 fi
 
+SSM_SERVICE_NAME="amazon-ssm-agent"
+SSM_BIN_NAME="amazon-ssm-agent"
+if [ "$(systemctl is-enabled snap.amazon-ssm-agent.amazon-ssm-agent.service)" == "enabled" ]; then
+    echo "Detected SSM agent installed via snap."
+    SSM_SERVICE_NAME="snap.amazon-ssm-agent.amazon-ssm-agent.service"
+    SSM_BIN_NAME="/snap/amazon-ssm-agent/current/amazon-ssm-agent"
+fi
+SSM_MANAGED_INSTANCE_ID=""
+
+check-option-value() {
+    if [ "${2:0:2}" == "--" ]; then
```

Comment:
is this meant to test that the 2nd parameter is not `--`?  If I do `./ecs-anywhere-install.sh --opt1 --opt2` I think this will complain, is that the intention?

Reply:
yes you're right, I think I put this check in a spot that is too broad. We shouldn't be checking on single args like --skip-registration, we be do want to check on args like `--cluster $CLUSTER_NAME`

Reply:
fixed